syncclient/client.py

```diff
@@ -249,10 +253,19 @@ def put_record(self, collection, record, **kwargs):
         Note that the server may impose a limit on the amount of data
         submitted for storage in a single BSO.
         """
+        if isinstance(record, six.string_types):
```

Comment:
We should add a note here stating that this shouldn't last once requests-hawk support for the JSON parameter is fixed.